admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -42,7 +42,15 @@ Assuming that your ownCloud installation is on Ubuntu, then the following comman
    
   # Install core packages
   sudo apt-get update -y
-  sudo apt-get install -y smbclient coreutils
+  sudo apt-get install -y smbclient php-smbclient coreutils
+  
+  # Enable the extension in your PHP installation
+  extension="smbclient.so"
```

Comment:
It seems that the text: "Enabling the extensions..." is now double present.
Here on line 47 and at line 57 (look at the current .rst file)
I guess L47 has been accidentially added

Ok for ``php-smbclient``

Reply:
Well, you're right, but it someone follows the first example - he has to enable the extension.

If he, on the other hand, follows the second or third method, he still has to enable this extension.

I think its better to have it in both code blocks, instead of requiring the user to look for the line he need in another code block.

Reply:
Independent where you get smbclient from, enable it or check that it is enabled.
For me clearly the last action point as described on L57 ff

Reply:
I suggest a middle ground, where we include the instruction to enable the module once, in a note admonition which should draw users' attention to it, after both sets of instructions.

Reply:
Sure, with a code box.